mongoose/notifications/NotificationSchema.ts

```diff
@@ -0,0 +1,33 @@
+import mongoose, { Schema } from 'mongoose';
+import MongooseException from '../../errors/MongooseException';
+import { NotificationType } from '../../models/notifications/NotificationType';
+import INotification from "../../models/notifications/INotification";
+import { formatJSON } from '../util/formatJSON';
+
+/**
+ * Mongoose schema for the notifications resource that takes an {@link INotifications} object. 
+ * The schema contains a user foreign key reference. All fields are required, and created/updated time stamps are added.
```

Comment:
Doc formatting should be:
@param {ObjectType} paramName general comments
Example: @param {NotificationType} type the type of notification; e.g. Follow or Message
@param {Bookean} read has the notification been read?